lib/base.js

```diff
@@ -7,12 +7,12 @@ var base = module.exports = Common.extend({
         if (node[0] === '#') {
           this.element = this._make('div',{
             id: node.replace(/^#/,''),
-            width: '100%'
+            "style": 'width: 100%;'
```

Comment:
width: 100% should be equivalent to style: 'width:100%', so lets keep it like before here